INFO     [🔧TEST CONFIG]:conftest.py:87 Loading test configuration
INFO     [🔧TEST CONFIG]:conftest.py:378 Navigating to: https://console.dev.apolo.us/
WARNING  [🔧TEST CONFIG]:conftest.py:420 [HTTP 200] GET https://api.dev.apolo.us/api/v1/config
Request Body:
<not available>
Response Body (first 1000 chars):
{"authorized": false, "auth_url": "https://auth.dev.apolo.us/authorize", "token_url": "https://auth.dev.apolo.us/oauth/token", "logout_url": "https://auth.dev.apolo.us/v2/logout?returnTo=https://console.dev.apolo.us", "client_id": "q3I0OzzGnTDkhRmpeJ7WWgaTCucmVxTL", "audience": "https://api.dev.apolo.us", "callback_urls": ["http://127.0.0.1:54540", "http://127.0.0.1:54541", "http://127.0.0.1:54542"], "headless_callback_url": "https://api.dev.apolo.us/oauth/show-code", "success_redirect_url": "https://api.dev.apolo.us/oauth/login"}
INFO     [🔧TEST CONFIG]:conftest.py:93 Creating data manager
INFO     [🔧TEST CONFIG]:conftest.py:123 Creating users manager
INFO     [🔧TEST CONFIG]:conftest.py:109 Creating API helper
INFO     [🔧TEST CONFIG]:conftest.py:117 Creating Apolo CLI instance
INFO     [🔧TEST CONFIG]:conftest.py:205 Continue using UserData(email='regression-a5dhoxuf8h@apolo.us', username='regression-a5dhoxuf8h', password='QCGd@^bk&9nE') as main test user...
INFO     [👤UsersManager]:users_manager.py:73 Setting main user to UserData(email='regression-a5dhoxuf8h@apolo.us', username='regression-a5dhoxuf8h', password='QCGd@^bk&9nE')
INFO     [🔧TEST CONFIG]:conftest.py:235 Continue using UserData(email='regression-gsy7qseopt@apolo.us', username='regression-gsy7qseopt', password='tPggZyb1q%&H') as second test user...
INFO     [👤UsersManager]:users_manager.py:82 Setting second user to UserData(email='regression-gsy7qseopt@apolo.us', username='regression-gsy7qseopt', password='tPggZyb1q%&H')
INFO     [🔧TEST CONFIG]:conftest.py:240 Continue using UserData(email='regression-pn7pzt1xoz@apolo.us', username='regression-pn7pzt1xoz', password='HFlSk%n4Y7GH') as third test user...
INFO     [👤UsersManager]:users_manager.py:91 Setting third user to UserData(email='regression-pn7pzt1xoz@apolo.us', username='regression-pn7pzt1xoz', password='HFlSk%n4Y7GH')
INFO     [📘TEST_INFO]:reporting.py:37 ▶️ STEP started: Verify Apolo CLI client is installed
INFO     [🖥apolo_CLI]:apolo_runner.py:25 check CLI version. Running command via cli:

apolo --version

INFO     [🖥apolo_CLI]:apolo_runner.py:86 check CLI version succeeded:

Apolo Platform Client 25.8.1

INFO     [📘TEST_INFO]:reporting.py:51 ✅ STEP completed: Verify Apolo CLI client is installed
INFO     [📘TEST_INFO]:reporting.py:86 ------------------------------------------------------------
INFO     [📘TEST_INFO]:reporting.py:87 📝 TEST started: User creates a second organization via CLI
INFO     [📘TEST_INFO]:reporting.py:88 ------------------------------------------------------------
INFO     [📘TEST_INFO]:reporting.py:37 ▶️ STEP started: Login via UI
INFO     [📘TEST_INFO]:reporting.py:37 ▶️ STEP started: Click login button
INFO     [🎭 Playwright]:base_page.py:16 <AuthPage>: Click log in button
INFO     [📘TEST_INFO]:reporting.py:51 ✅ STEP completed: Click login button
INFO     [📘TEST_INFO]:reporting.py:225 📸 Screenshot captured and attached (success): /home/runner/work/apolo-test-automation/apolo-test-automation/reports/screenshots/ui_click_login_button_success_311dc1.png
INFO     [📘TEST_INFO]:reporting.py:37 ▶️ STEP started: Enter email
INFO     [🎭 Playwright]:base_page.py:16 <LoginPage>: Enter regression-a5dhoxuf8h@apolo.us email
INFO     [📘TEST_INFO]:reporting.py:51 ✅ STEP completed: Enter email
INFO     [📘TEST_INFO]:reporting.py:225 📸 Screenshot captured and attached (success): /home/runner/work/apolo-test-automation/apolo-test-automation/reports/screenshots/ui_enter_email_success_ec5e60.png
INFO     [📘TEST_INFO]:reporting.py:37 ▶️ STEP started: Enter password
INFO     [🎭 Playwright]:base_page.py:16 <LoginPage>: Enter ********* password
INFO     [📘TEST_INFO]:reporting.py:51 ✅ STEP completed: Enter password
INFO     [📘TEST_INFO]:reporting.py:225 📸 Screenshot captured and attached (success): /home/runner/work/apolo-test-automation/apolo-test-automation/reports/screenshots/ui_enter_password_success_0c91e1.png
INFO     [📘TEST_INFO]:reporting.py:37 ▶️ STEP started: Click Continue button
INFO     [🎭 Playwright]:base_page.py:16 <LoginPage>: Click continue button
WARNING  [🔧TEST CONFIG]:conftest.py:420 [HTTP 200] GET https://api.dev.apolo.us/api/v1/config
Request Body:
<not available>
Response Body (first 1000 chars):
{"authorized": true, "orgs": [], "clusters": [], "projects": [], "admin_url": "https://api.dev.apolo.us/apis/admin/v1", "auth_url": "https://auth.dev.apolo.us/authorize", "token_url": "https://auth.dev.apolo.us/oauth/token", "logout_url": "https://auth.dev.apolo.us/v2/logout?returnTo=https://console.dev.apolo.us", "client_id": "q3I0OzzGnTDkhRmpeJ7WWgaTCucmVxTL", "audience": "https://api.dev.apolo.us", "callback_urls": ["http://127.0.0.1:54540", "http://127.0.0.1:54541", "http://127.0.0.1:54542"], "headless_callback_url": "https://api.dev.apolo.us/oauth/show-code", "success_redirect_url": "https://api.dev.apolo.us/oauth/login"}
WARNING  [🔧TEST CONFIG]:conftest.py:420 [HTTP 200] GET https://api.dev.apolo.us/api/v1/users/regression-a5dhoxuf8h/permissions?uri=blob:
Request Body:
<not available>
Response Body (first 1000 chars):
[]
WARNING  [🔧TEST CONFIG]:conftest.py:420 [HTTP 200] GET https://api.dev.apolo.us/apis/admin/v1/invites/incoming
Request Body:
<not available>
Response Body (first 1000 chars):
[]
WARNING  [🔧TEST CONFIG]:conftest.py:420 [HTTP 200] GET https://api.dev.apolo.us/apis/admin/v1/users/regression-a5dhoxuf8h?include=orgs
Request Body:
<not available>
Response Body (first 1000 chars):
{"name": "regression-a5dhoxuf8h", "email": "regression-a5dhoxuf8h@apolo.us", "first_name": null, "last_name": null, "created_at": "2025-09-19T01:32:57.543342+00:00", "terms_accepted": true, "orgs": []}
INFO     [🎭 Playwright]:base_page.py:16 <LoginPage>: Waiting for spinner
INFO     [🎭 Playwright]:base_page.py:16 <LoginPage>: Waiting for spinner to disappear
WARNING  [🔧TEST CONFIG]:conftest.py:420 [HTTP 200] GET https://api.dev.apolo.us/apis/admin/v1/users/regression-a5dhoxuf8h?include=clusters&include=system_role
Request Body:
<not available>
Response Body (first 1000 chars):
{"name": "regression-a5dhoxuf8h", "email": "regression-a5dhoxuf8h@apolo.us", "first_name": null, "last_name": null, "created_at": "2025-09-19T01:32:57.543342+00:00", "terms_accepted": true, "clusters": []}
INFO     [📘TEST_INFO]:reporting.py:51 ✅ STEP completed: Click Continue button
INFO     [📘TEST_INFO]:reporting.py:225 📸 Screenshot captured and attached (success): /home/runner/work/apolo-test-automation/apolo-test-automation/reports/screenshots/ui_click_continue_button_success_6c4a3e.png
INFO     [📘TEST_INFO]:reporting.py:37 ▶️ STEP started: Verify that Welcome new user page displayed
INFO     [🎭 Playwright]:base_page.py:16 <WelcomeNewUserPage>: Check if page loaded
INFO     [📘TEST_INFO]:reporting.py:51 ✅ STEP completed: Verify that Welcome new user page displayed
INFO     [📘TEST_INFO]:reporting.py:225 📸 Screenshot captured and attached (success): /home/runner/work/apolo-test-automation/apolo-test-automation/reports/screenshots/verify_ui_page_displayed_success_61d01e.png
INFO     [📘TEST_INFO]:reporting.py:51 ✅ STEP completed: Login via UI
INFO     [📘TEST_INFO]:reporting.py:225 📸 Screenshot captured and attached (success): /home/runner/work/apolo-test-automation/apolo-test-automation/reports/screenshots/ui_login_success_1ca24b.png
INFO     [📘TEST_INFO]:reporting.py:37 ▶️ STEP started: Login to apolo CLI with access token
INFO     [🖥apolo_CLI]:apolo_runner.py:25 login-with-token. Running command via cli:

apolo config login-with-token eyJhbGciOiJSUzI1NiIsInR5cCI6IkpXVCIsImtpZCI6IjhyNXZCR0E4ZXNuc1IzbXJ0NUFSViJ9.eyJodHRwczovL3BsYXRmb3JtLm5ldXJvbWF0aW9uLmlvL3VzZXIiOiJyZWdyZXNzaW9uLWE1ZGhveHVmOGgiLCJodHRwczovL3BsYXRmb3JtLm5ldXJvbWF0aW9uLmlvL2xvZ2luX3JlZGlyZWN0X3VybCI6Imh0dHBzOi8vY29uc29sZS5kZXYuYXBvbG8udXMiLCJodHRwczovL3BsYXRmb3JtLmFwb2xvLnVzL29yaWdpbiI6Imh0dHBzOi8vY29uc29sZS5kZXYuYXBvbG8udXMiLCJpc3MiOiJodHRwczovL2F1dGguZGV2LmFwb2xvLnVzLyIsInN1YiI6ImF1dGgwfDY4Y2NiMmJmZDJjNjRhODYyMmNhMmM3NSIsImF1ZCI6WyJodHRwczovL2FwaS5kZXYuYXBvbG8udXMiLCJodHRwczovL2Rldi1hcG9sby11cy51cy5hdXRoMC5jb20vdXNlcmluZm8iXSwiaWF0IjoxNzU4MjQ4OTkzLCJleHAiOjE3NTgzMzUzOTMsInNjb3BlIjoib3BlbmlkIHByb2ZpbGUgZW1haWwiLCJhenAiOiJxM0kwT3p6R25URGtoUm1wZUo3V1dnYVRDdWNtVnhUTCJ9.SMRFYq_wpIgOICh8dYhvvyDHYBNl5OhwAKmZZ6WJe4KLSEpK5UiI8HIkhMPV4W-_UtMZpMnWOz-jbgMmLuholKSgYrjbhAv52wCxVubXYiq7rn1w9I6AiHguEAo-m19V_r0Rq64isWsIZnIcTNLyut6JXHuYVOnepxJ7HHlnKp9F8c-4xr1ZbY4mTk8YOnp29VWylZwH4crUo2FkJowvBPsSQyUUhHj_etvmeZRpRZ79bFjQL5r8QXArvmXB85nYK418sGaGr4aag6BlrASIw9IDIm9DFyWi_E5QVJaBtUIh5G_yKcnZh80dEFcNDVuG5RVjbtAun2xgrXniGi8lfg https://api.dev.apolo.us/api/v1

INFO     [🖥apolo_CLI]:apolo_runner.py:86 login-with-token succeeded:

You were successfully logged out.
Logged into https://api.dev.apolo.us/api/v1 as regression-a5dhoxuf8h
Read the docs at https://docs.apolo.us or run `apolo --help` to see the reference

INFO     [📘TEST_INFO]:reporting.py:51 ✅ STEP completed: Login to apolo CLI with access token
INFO     [📘TEST_INFO]:reporting.py:37 ▶️ STEP started: Verify organization_count
INFO     [🖥apolo_CLI]:apolo_runner.py:25 get organizations. Running command via cli:

apolo admin get-orgs

INFO     [🖥apolo_CLI]:apolo_runner.py:86 get organizations succeeded:

Name  
 ━━━━━━

INFO     [🖥apolo_CLI]:apolo_admin.py:163 Fetched organizations: []
INFO     [📘TEST_INFO]:reporting.py:51 ✅ STEP completed: Verify organization_count
INFO     [📘TEST_INFO]:reporting.py:37 ▶️ STEP started: Add new organization via apolo CLI
INFO     [🔧DATA MANAGER]:test_data.py:56 Added organization: org_name=regression-org-pxixvqz6452m, gherkin_name=My-organization
INFO     [🖥apolo_CLI]:apolo_runner.py:25 create organization 'regression-org-pxixvqz6452m'. Running command via cli:

apolo admin add-org regression-org-pxixvqz6452m

ERROR    [🖥apolo_CLI]:apolo_runner.py:82 create organization 'regression-org-pxixvqz6452m' failed:

ERROR: There are no clusters available. Please run 'apolo logout' and then 'apolo login'.
Full logs are available under /home/runner/.apolo/logs/apolo-run-2025-09-19_02-30-01.txt

INFO     [🖥apolo_CLI]:apolo_runner.py:25 login-with-token. Running command via cli:

apolo config login-with-token eyJhbGciOiJSUzI1NiIsInR5cCI6IkpXVCIsImtpZCI6IjhyNXZCR0E4ZXNuc1IzbXJ0NUFSViJ9.eyJodHRwczovL3BsYXRmb3JtLm5ldXJvbWF0aW9uLmlvL3VzZXIiOiJyZWdyZXNzaW9uLWE1ZGhveHVmOGgiLCJodHRwczovL3BsYXRmb3JtLm5ldXJvbWF0aW9uLmlvL2xvZ2luX3JlZGlyZWN0X3VybCI6Imh0dHBzOi8vY29uc29sZS5kZXYuYXBvbG8udXMiLCJodHRwczovL3BsYXRmb3JtLmFwb2xvLnVzL29yaWdpbiI6Imh0dHBzOi8vY29uc29sZS5kZXYuYXBvbG8udXMiLCJpc3MiOiJodHRwczovL2F1dGguZGV2LmFwb2xvLnVzLyIsInN1YiI6ImF1dGgwfDY4Y2NiMmJmZDJjNjRhODYyMmNhMmM3NSIsImF1ZCI6WyJodHRwczovL2FwaS5kZXYuYXBvbG8udXMiLCJodHRwczovL2Rldi1hcG9sby11cy51cy5hdXRoMC5jb20vdXNlcmluZm8iXSwiaWF0IjoxNzU4MjQ4OTkzLCJleHAiOjE3NTgzMzUzOTMsInNjb3BlIjoib3BlbmlkIHByb2ZpbGUgZW1haWwiLCJhenAiOiJxM0kwT3p6R25URGtoUm1wZUo3V1dnYVRDdWNtVnhUTCJ9.SMRFYq_wpIgOICh8dYhvvyDHYBNl5OhwAKmZZ6WJe4KLSEpK5UiI8HIkhMPV4W-_UtMZpMnWOz-jbgMmLuholKSgYrjbhAv52wCxVubXYiq7rn1w9I6AiHguEAo-m19V_r0Rq64isWsIZnIcTNLyut6JXHuYVOnepxJ7HHlnKp9F8c-4xr1ZbY4mTk8YOnp29VWylZwH4crUo2FkJowvBPsSQyUUhHj_etvmeZRpRZ79bFjQL5r8QXArvmXB85nYK418sGaGr4aag6BlrASIw9IDIm9DFyWi_E5QVJaBtUIh5G_yKcnZh80dEFcNDVuG5RVjbtAun2xgrXniGi8lfg https://api.dev.apolo.us/api/v1

INFO     [🖥apolo_CLI]:apolo_runner.py:86 login-with-token succeeded:

You were successfully logged out.
Logged into https://api.dev.apolo.us/api/v1 as regression-a5dhoxuf8h, current cluster is default, org is regression-org-pxixvqz6452m project is None
Read the docs at https://docs.apolo.us or run `apolo --help` to see the reference

INFO     [📘TEST_INFO]:reporting.py:51 ✅ STEP completed: Add new organization via apolo CLI
INFO     [📘TEST_INFO]:reporting.py:37 ▶️ STEP started: Verify organization_count
INFO     [🖥apolo_CLI]:apolo_runner.py:25 get organizations. Running command via cli:

apolo admin get-orgs

INFO     [🖥apolo_CLI]:apolo_runner.py:86 get organizations succeeded:

Name                         
 ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 
  regression-org-pxixvqz6452m

INFO     [🖥apolo_CLI]:apolo_admin.py:163 Fetched organizations: ['regression-org-pxixvqz6452m']
INFO     [📘TEST_INFO]:reporting.py:51 ✅ STEP completed: Verify organization_count
INFO     [📘TEST_INFO]:reporting.py:37 ▶️ STEP started: Verify organization is listed in the CLI output
INFO     [📘TEST_INFO]:reporting.py:51 ✅ STEP completed: Verify organization is listed in the CLI output
INFO     [📘TEST_INFO]:reporting.py:37 ▶️ STEP started: Add new organization via apolo CLI
INFO     [🔧DATA MANAGER]:test_data.py:56 Added organization: org_name=regression-org-sq0pcp7dzblo, gherkin_name=Second-organization
INFO     [🖥apolo_CLI]:apolo_runner.py:25 create organization 'regression-org-sq0pcp7dzblo'. Running command via cli:

apolo admin add-org regression-org-sq0pcp7dzblo

INFO     [🖥apolo_CLI]:apolo_runner.py:86 create organization 'regression-org-sq0pcp7dzblo' succeeded:



INFO     [📘TEST_INFO]:reporting.py:51 ✅ STEP completed: Add new organization via apolo CLI
INFO     [📘TEST_INFO]:reporting.py:37 ▶️ STEP started: Verify organization_count
INFO     [🖥apolo_CLI]:apolo_runner.py:25 get organizations. Running command via cli:

apolo admin get-orgs

INFO     [🖥apolo_CLI]:apolo_runner.py:86 get organizations succeeded:

Name                         
 ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 
  regression-org-khuxse1d4d2y

INFO     [🖥apolo_CLI]:apolo_admin.py:163 Fetched organizations: ['regression-org-khuxse1d4d2y']
ERROR    [📘TEST_INFO]:reporting.py:56 ❌ STEP failed: Verify organization_count
ERROR    [📘TEST_INFO]:reporting.py:95 ❌ TEST failed: User creates a second organization via CLI
Exception: [AssertionError] during step: Verify organization_count
Message: Expected 2 organizations, got 1
INFO     [📘TEST_INFO]:reporting.py:98 ------------------------------------------------------------
INFO     [🌐API_helper]:api_helper.py:54 GET https://api.dev.apolo.us/apis/admin/v1/orgs
INFO     [🌐API_helper]:api_helper.py:165 Status: 200. Response: [{'name': 'regression-org-pxixvqz6452m', 'balance': {'credits': '500', 'spent_credits': '0'}, 'user_default_credits': None, 'notification_intervals': {'balance_projection_seconds': [604800, 259200, 86400], 'balance_amount': [0], 'balance_depletion_seconds': [86400, 604800]}}, {'name': 'regression-org-sq0pcp7dzblo', 'balance': {'credits': '500', 'spent_credits': '0'}, 'user_default_credits': None, 'notification_intervals': {'balance_projection_seconds': [604800, 259200, 86400], 'balance_amount': [0], 'balance_depletion_seconds': [86400, 604800]}}]
INFO     [🔧TEST CONFIG]:conftest.py:291 Cleaning up 2 organisations
INFO     [🌐API_helper]:api_helper.py:172 Delete organization response: <ClientResponse(https://api.dev.apolo.us/apis/admin/v1/orgs/regression-org-pxixvqz6452m) [200 OK]>
<CIMultiDictProxy('Access-Control-Allow-Credentials': 'true', 'Content-Length': '274', 'Content-Type': 'application/json; charset=utf-8', 'Date': 'Fri, 19 Sep 2025 02:30:51 GMT', 'Server': 'Python/3.13 aiohttp/3.12.15', 'Vary': 'Origin', 'x-service-version': 'platform-admin/0.0.0.post1.dev0+4cdab15', 'Via': '1.1 google', 'Alt-Svc': 'h3=":443"; ma=2592000,h3-29=":443"; ma=2592000')>

INFO     [🔧TEST CONFIG]:conftest.py:316 Removed organisation: regression-org-pxixvqz6452m
INFO     [🌐API_helper]:api_helper.py:172 Delete organization response: <ClientResponse(https://api.dev.apolo.us/apis/admin/v1/orgs/regression-org-sq0pcp7dzblo) [200 OK]>
<CIMultiDictProxy('Access-Control-Allow-Credentials': 'true', 'Content-Length': '274', 'Content-Type': 'application/json; charset=utf-8', 'Date': 'Fri, 19 Sep 2025 02:30:52 GMT', 'Server': 'Python/3.13 aiohttp/3.12.15', 'Vary': 'Origin', 'x-service-version': 'platform-admin/0.0.0.post1.dev0+4cdab15', 'Via': '1.1 google', 'Alt-Svc': 'h3=":443"; ma=2592000,h3-29=":443"; ma=2592000')>

INFO     [🔧TEST CONFIG]:conftest.py:316 Removed organisation: regression-org-sq0pcp7dzblo
INFO     [🔧TEST CONFIG]:conftest.py:321 Closing all browser sessions
INFO     [🔧TEST CONFIG]:conftest.py:327 Stopped Playwright
WARNING  [🔧TEST CONFIG]:conftest.py:337 Failed to close context: BrowserContext.close: Target page, context or browser has been closed
INFO     [🔧TEST CONFIG]:conftest.py:342 Closed browser (post-stop)
INFO     [🔧TEST CONFIG]:conftest.py:347 Browser cleanup finished